mlflow/store/tracking/__init__.py

```diff
@@ -10,5 +10,6 @@
 # Also used as default location for artifacts, when not provided, in non local file based backends
 # (eg MySQL)
 DEFAULT_LOCAL_FILE_AND_ARTIFACT_PATH = "./mlruns"
+DEFAULT_ARTIFACTS_URI = "mlflow-artifacts:/"
```

Comment:
Can we add an inline comment explaining what this is used for?

Reply:
certainly!